pkg/term/tc_solaris_cgo.go

```diff
@@ -0,0 +1,63 @@
+// +build solaris,cgo
+
+package term
+
+import (
+	"syscall"
+	"unsafe"
+)
+
+// #include <termios.h>
+import "C"
+
+// Termios is the Unix API for terminal I/O.
+// It is passthgrouh for syscall.Termios in order to make it portable with
```

Comment:
Ah! Was a typo in tc_linux_cgo.go as well. Fixed both and updated PR. Thanks @thaJeztah !